monotonic_test.go

```diff
@@ -0,0 +1,118 @@
+package ulid
+
+import (
+	"bytes"
+	"encoding/hex"
+	"math/rand"
+	"reflect"
+	"sort"
+	"testing"
+	"time"
+)
+
+func TestMonotonic(t *testing.T) {
+	for i := int64(0); i < 1000; i++ {
```

Comment:
Instead of a unit test, I suggest we cover the `Monotonicity` property by a property based test like we're doing for other properties such as [lexicographical order being equivalent to byte order](https://github.com/oklog/ulid/blob/master/ulid_test.go#L193).

Reply:
That's a good idea. When I have a better idea of what the API should look like, I'll rework the tests.